task-maker-format/src/ioi/format/italian_yaml/cases_gen.rs

```diff
@@ -521,17 +521,18 @@ where
                 self.subtask_id, score
             )
         })?;
-        let description = if line.len() >= 2 {
-            Some(line[1].as_str().to_string())
+        let name = if line.len() >= 2 {
+            // Remove whitespaces for retrocompatibility with descriptions
+            let s = line[1].as_str();
+            Some(s.chars().filter(|&c| c != ' ' && c != '\t').collect())
```

Comment:
```suggestion
            Some(s.chars().filter(|&c| c != ' ' && c != '\t' && c != '#').collect())
```

If I read the grammar correctly, `#` is also not allowed in subtask names: https://github.com/edomora97/task-maker-rust/blob/master/task-maker-format/src/ioi/format/italian_yaml/GEN.pest#L12

Reply:
I don't think `#` is allowed in the task description.

Reply:
Fair enough